lib/bundler/gem_helper.rb

```diff
@@ -83,7 +83,7 @@ def build_gem
 
     def install_gem(built_gem_path = nil, local = false)
       built_gem_path ||= build_gem
-      out, _ = sh_with_code("gem install '#{built_gem_path}'#{" --local" if local}")
+      out, _code = sh_with_code("gem install '#{built_gem_path}'#{" --local" if local}")
```

Comment:
Since this was the only warning for `Style/TrailingUnderscoreVariable`, I just dropped that commit for now.